### PR TITLE
Generalize __get_namespace to allow checking for other arguments

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

File: pkgdb2client/cli.py [hunk 1]
```diff
@@ -51,14 +51,14 @@ class ActionError(Exception):
...
-def __get_namespace(args):
+def __get_namespace(args, kw='package'):
     ''' Return the namespace of the package as specified as an argument
     unless it was specified in the package name.
     '''
     namespace = args.namespace
-    if hasattr(args, 'package'):
-        if args.package and '/' in args.package:
-            namespace = args.package.split('/', 1)[0]
+    if hasattr(args, kw):
```
Comment (sayanchowdhury) [inline pkgdb2client/cli.py]: A small suggestion here: I have recently stopped using `hasattr` in favor of `try-except` after reading this - https://hynek.me/articles/hasattr/

Comment (pypingou) [inline pkgdb2client/cli.py]: Nice reading, thanks for sharing

**Comment by pypingou** [comment]
@sayanchowdhury updated :)

**Comment by sayanchowdhury** [comment]
Looks good :+1:

**Pull request merged**